### ISSUE TITLE
Improve browser support by identifying and adding polyfills for Javascript errors

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#1511]
The app has lots of noise on Sentry from various Javascript features that are not supported on specific browsers and/or operating systems. These are hard to fix without knowing where exactly they occur and how to replicate them.

If you find a Javascript error in your console, report it in a comment below, along with a description of the effects and how to reproduce it.

Then look for polyfills or shims, or just changes to the code, that could be used to add support / fix the problem.

**Comment by AmitJoki** [comment]
Is this issue still relevant?

**Comment by ragesoss** [comment]
Yes, I think so... especially if anyone happens to try it with an older browser or something outside of the usual Firefox/Chrome/Safari/Edge

**Comment by AmitJoki** [comment]
How old are we supposed to go? Because if searched extensively, there might be polyfills even for the earliest of the IEs. Is there a minimum threshold of versions that we would like to be supported on?

Do we have any data on this? Like visits per browser. That will help narrow down the scope of this issue and I can draft a GCI task based on this.

**Comment by ragesoss** [comment]
@AmitJoki at this point, we "officially" are trying to support  everything except IE. I think all, or nearly all, versions of IE are either completely non-functional or have significant problems. But it's more a matter of doing the best we can with commonly used browsers. In particular, getting better for older Safaris, Chromes, and figuring out what versions of Opera do or don't work would all be nice.
